### DISCUSSION TITLE
Model Saving and Persistence AutoGluon

### DISCUSSION THREAD

**Discussion by alibek-wand** [autogluon/autogluon#2247] [Q&A] [answered]
Hi all,

When the AutoGluon model is saved (using save()) is persistence guaranteed? I mean when new version of AutoGluon is released, can I still load this old model with load() function? If not, what can I do?

Is there Slack or Telegram channel for this kind of quick Q&A?

Thanks)

**Comment by Innixma** [comment, marked as answer]
You have to load the model with the same AutoGluon version and dependency versions as you saved it with. Cross-version loading is not supported and will not work. Either 1. re-train with the new version or 2. keep using the old version to load an old model artifact for inference.

Re slack/telegram: We are planning to create one later this year, it will be announced on our twitter + GitHub
